vector-im/element-android
# [Device management] Session details screen (PSG-686)

## Type of change


## Content


Adding a new screen to show session details. It can be reached from the session overview screen. For now, we only have sections for non-extended data - session name, device id, IP, latest activity.

## Motivation and context


Closes #7077 

## Screenshots / GIFs




<img src="https://user-images.githubusercontent.com/46314705/189354523-bf399cd3-e297-4e1e-89d4-62745c32faeb.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/189354778-94ac1f9a-ce90-4a10-b88a-8f0799c6a907.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/189354517-b3a8f429-e9b9-4f6d-96f4-099fecb583d0.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/189354782-8a9f6a57-0d80-4fa8-98fa-3d03bcebf380.png" width=300 />

<img src="https://user-images.githubusercontent.com/46314705/189354510-de79215a-9c53-4b50-b919-d70c14782080.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/189354783-ad0af3d5-a68a-4553-bc12-a881de987ad1.png" width=300 />

## Tests



- Enable the new device management feature flag
- Go to Settings -> Security & Privacy -> Show all sessions (V2, WIP)
- Press the current session card
- Press the Session details entry
- Check a new screen is displayed with the correct information

## Tested devices

- OS version(s): Android 11

## Checklist
